west.yml

```diff
@@ -210,7 +210,7 @@ manifest:
         - hal
     - name: hal_renesas
       path: modules/hal/renesas
-      revision: b4fe8925d95bddc27e6d1666890f560d30cf9166
+      revision: pull/17/head
```

Comment:
Please add [DNM] at the beginning of your commit message.
[manifest: Correct commit ID for hal_renesas](https://github.com/zephyrproject-rtos/zephyr/pull/72150/commits/be13c606e6374824d274548e6524d50f93c315c1)

You will rewrite this commit after hal_renesas is updated.